src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -204,14 +235,23 @@ export class WorkItemDetailComponent implements OnInit {
     }    
   }
 
-  save(): void {
-    this.workItemService
-      .update(this.workItem)
-      .then((workItem) => {
-        this.workItem.attributes['version'] = workItem.attributes['version'];
-        this.activeOnList();          
-    });
-     
+  save(): void {        
+    if (this.workItem.id){
+      this.workItemService
+        .update(this.workItem)
+        .then((workItem) => {
+          this.workItem.attributes['version'] = workItem.attributes['version'];
+          this.activeOnList();          
+      });
+    }else{
+      if (this.workItem.attributes['system.title']){
```

Comment:
Can you use **this.validTitle** flag here. So that if you write some special validation logic tomorrow in **isValid()** function, then that'll work here as well.